astronomer/providers/amazon/aws/triggers/batch.py

```diff
@@ -104,3 +105,63 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 yield TriggerEvent({"status": "error", "message": error_message})
         except Exception as e:
             yield TriggerEvent({"status": "error", "message": str(e)})
+
+
+class BatchSensorTrigger(BaseTrigger):
+    """
+    Checks for the state of a submitted job_id to AWS Batch is completed or not.
+    BatchSessorTrigger is fired as deferred class with params to poll the job state in Triggerer
+
+    :param job_id: the job ID, to poll for job completion or not
+    :param aws_conn_id: connection id of AWS credentials / region name. If None,
+        credential boto3 strategy will be used.
+    :param region_name: AWS region name to use .
+        Override the region_name in connection (if provided)
+    """
+
+    def __init__(
+        self,
+        job_id: str,
+        region_name: Optional[str],
+        aws_conn_id: Optional[str] = "aws_default",
+        polling_period_seconds: float = 5,
```

Comment:
`polling_period_seconds` is missing in docstring and what about renaming it to `poll_interval`?